great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -107,9 +107,12 @@ def to_dict(self) -> dict:
         """
         return asdict(self)
 
+    @public_api
```

Comment:
Loosk like this is on RuleStats - should it be on DataAssistantResult instead (looking at PR title)